lib/Conversion/TorchToLinalg/Linear.cpp

```diff
@@ -803,6 +803,8 @@ class ConvertAtenConvolutionOp : public OpConversionPattern<AtenConvolutionOp> {
       inputZp = typeConverter->materializeTargetConversion(
           rewriter, loc, typeConverter->convertType(inputZp.getType()),
           inputZp);
+      inputZp =
+          rewriter.create<arith::TruncIOp>(loc, rewriter.getI32Type(), inputZp);
```

Comment:
How did we decide on whether to keep it i32 or i64?

Reply:
This is the accumulator type for i8 quantized convolutions. I haven't seen any other kinds of quantized convs yet.

Reply:
Here's a link to the linalg op generating code:

<https://github.com/llvm/llvm-project/blob/c6b6e18c4d25305ab98b6eab752de99ea4e15344/mlir/python/mlir/dialects/linalg/opdsl/ops/core_named_ops.py#L859>